concent_api/core/message_handlers.py

```diff
@@ -1231,19 +1235,20 @@ def set_subtask_messages(
 
 
 def store_message(
-    golem_message:          message.base.Message,
-    task_id:                str,
-    subtask_id:             str,
+    golem_message: message.base.Message,
+    task_id: str,
+    subtask_id: str,
 ) -> StoredMessage:
     assert golem_message.header.type_ in library
 
     message_timestamp = parse_timestamp_to_utc_datetime(golem_message.timestamp)
     stored_message = StoredMessage(
-        type        = golem_message.header.type_,
-        timestamp   = message_timestamp,
-        data        = copy.copy(golem_message).serialize(),
-        task_id     = task_id,
-        subtask_id  = subtask_id,
+        type=golem_message.header.type_,
+        timestamp=message_timestamp,
+        data=copy.copy(golem_message).serialize(),
```

Comment:
I remember there were some issues related to signatures when doing this without `copy`. I am not sure if they were resolved. Maybe it would be better to check this in separate issue.

Reply:
I think you are right, there was something with signatures. By serializing message was signed and serializing the same message (already signed) was cause of errors. In this PR I only reformated this to our new coding style. What do you think? Should it be investigated?

Reply:
And also if this is needed here please change import from:
```python
from copy import copy
```
than
```python
import copy
```
And I'm agree with you guys that it should be checked